### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by harveyjavier**
- Update Crypto Loco and SparkPoint Wallet App availability on iOS platform
- Add terms and conditions for SparkDeFi Launchpad

### CODE REVIEW

**Review by swengr-janan — COMMENTED** [review]
Updated!

**Review by kojiadrianojr — APPROVED** [review]
Updatified!

File: crypto-loco/index.html [hunk 1]
```diff
@@ -98,13 +98,13 @@
...
     <p>
       At some point, we may wish to update the app. The app is
-      currently available on Android – the requirements for
-      system(and for any additional systems we
+      currently available on Android &amp; iOS – the requirements for
+      both systems(and for any additional systems we
       decide to extend the availability of the app to) may change,
       and you’ll need to download the updates if you want to keep
-      using the app. SparkPoint Technologies does not promise that it
+      using the app. SparkPoint Technologies, Inc. does not promise that it
```
Comment (Pseudoman21) [inline crypto-loco/index.html:105]: Di na po to Kuya papalitan ki Innovations? hehe

File: wallet/index.html [hunk 2]
```diff
@@ -50,10 +50,22 @@
...
-      download the updates if you want to keep using the app. SparkPoint Technologies, Inc. does not promise that it will always update the app so that it is relevant to you and/or works with the Android version that you have installed on
-      your device. However, you promise to always accept updates to the application when offered to you, We may also wish to stop providing the app, and may terminate use of it at any time without giving notice of termination to you. Unless
-      we tell you otherwise, upon any termination, (a) the rights and licenses granted to you in these terms will end; (b) you must stop using the app, and (if needed) delete it from your device.
+      At some point, we may wish to update the app. The app is
+      currently available on Android &amp; iOS – the requirements for
+      both systems(and for any additional systems we
+      decide to extend the availability of the app to) may change,
+      and you’ll need to download the updates if you want to keep
+      using the app. SparkPoint Technologies, Inc. does not promise that it
```
Comment (Pseudoman21) [inline wallet/index.html:58]: Tapos ito po. hehe

**Comment by crs26** [comment]
:rocket:

**Comment by swengr-janan** [comment]
Ay nice @Pseudoman21 !! Missed it. Hehehe suri my bad

**Comment by Pseudoman21** [comment]
> Ay nice @Pseudoman21 !! Missed it. Hehehe suri my bad

Ox lang po Kuya haha naagihan lang po 😅

**Comment by swengr-janan** [comment]
Ay wait, tama na baga jeyps? SparkPoint Technologies, Inc.

**Comment by Pseudoman21** [comment]
Ay awu po palan hahahaha

**Comment by harveyjavier** [comment]
May hangover pa si jeypz sa SparkPoint Innovations, Inc. haha

**Comment by Pseudoman21** [comment]
> May hangover pa si jeypz sa SparkPoint Innovations, Inc. haha

hahaha nalula po 😆

**Comment by swengr-janan** [comment]
Hahahaha pati ako nagkarilibong na man sa lula.

**Pull request merged**